pkgs/applications/virtualization/crun/default.nix

```diff
@@ -1,19 +1,16 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook, go-md2man, pkgconfig
+{ stdenv, lib, fetchurl, go-md2man, pkgconfig
 , libcap, libseccomp, python3, systemd, yajl }:
 
 stdenv.mkDerivation rec {
   pname = "crun";
-  version = "0.10.6";
+  version = "0.11";
 
-  src = fetchFromGitHub {
-    owner = "containers";
-    repo = pname;
-    rev = version;
-    sha256 = "0v1hrlpnln0c976fb0k2ig4jv11qbyzf95z0wy92fd8r8in16rc1";
-    fetchSubmodules = true;
+  src = fetchurl {
+    url = "https://github.com/containers/crun/releases/download/${version}/crun-${version}.tar.gz";
+    sha256 = "0xn95pqd49wbqh34din5psqvlhdil5m0p3aj3l06zrs3llan54v6";
```

Comment:
any particular reason to not use source distribution?

Reply:
Using `fetchFromGitHub` needs `fetchSubmodules`, `leaveDotGit` to build and it still doesn't set the version correctly. Using the release tarball seems easier.

Reply:
ah, do you mind adding a comment so that someone else isn't tempted to try and use source then :)

Reply:
Updated with comment.